karuta.js

```diff
@@ -0,0 +1,73 @@
+import { CSV } from "https://js.sabae.cc/CSV.js";
+import { shuffle } from "https://js.sabae.cc/shuffle.js";
+
+window.showHint = () => {
+  alert(currentKaruta.origin_emblem_text);
+}
+
+window.showAnswer = () => {
+  const targetKaruta = document.getElementById(currentKaruta.id);
+  targetKaruta.border = 10;
+}
+
+window.drawNextKaruta = () => {
+  disableDraw = true;
+  toggleBtn();
+  const drawnKaruta = data.pop();
+  questionText.textContent = drawnKaruta.name_ja;
+  currentKaruta = drawnKaruta;
+  if (data.length == 0) {
+    if(wrongCount == 0) {
+      alert("素晴らしい！君は高専博士だ！");
+    } else if (wrongCount > 1 && wrongCount < 10) {
+      alert("すごい！君は高専マニアだ！");
+    } else if (wrongCount > 11 && wrongCount < 20) {
+      alert("いいね！君は高専人だ！");
+    } else {
+      alert("お疲れ様。それぞれの高専のことを覚えてくれたかな？");
```

Comment:
この場合、`wrongCount` が 1 の時、10, 11 の時に else に入ってしまいます。
いろんな区間の取り方があると思いますが、閉区間にする例を以下に示しておきます。
```suggestion
    if(wrongCount == 0) {
      alert("素晴らしい！君は高専博士だ！");
    } else if (wrongCount >= 1 && wrongCount <= 10) {
      alert("すごい！君は高専マニアだ！");
    } else if (wrongCount >= 11 && wrongCount <= 20) {
      alert("いいね！君は高専人だ！");
    } else {
      alert("お疲れ様。それぞれの高専のことを覚えてくれたかな？");
```